sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/CommunicationRoom.java

```diff
@@ -27,6 +28,24 @@ public CommunicationRoom(String roomId, OffsetDateTime validFrom, OffsetDateTime
         this.validFrom = validFrom;
         this.validUntil = validUntil;
         this.createdAt = createdAt;
+        this.isPstnDialOutEnabled = false;
```

Comment:
This method should just call the other overload.

```
this(roomId, validFrom, validUntil, createdAt, null);
```

Reply:
Also, we should not set this to `false` and leave it as `null`. This is a response type and the service should return this value and we should not infer the default in the client.